roles/openshift_health_checker/openshift_checks/package_update.py

```diff
@@ -9,6 +9,24 @@ class PackageUpdate(NotContainerizedMixin, OpenShiftCheck):
     name = "package_update"
     tags = ["preflight"]
 
+    supported = ["Red Hat Enterprise Linux"]
+
     def run(self, tmp, task_vars):
         args = {"packages": []}
-        return self.module_executor("check_yum_update", args, tmp, task_vars)
+
+        os = self.get_operating_system(task_vars)
+        if os and self.is_os_supported(os):
+            return self.module_executor("check_yum_update", args, tmp, task_vars)
+
+        return {"changed": False, "msg": "Skipping check on unsupported OS: {os}".format(os=os)}
+
+    def get_operating_system(self, task_vars):
+        info = self.module_executor("docker_info", {}, task_vars)
```

Comment:
@sosiouxme per your comment [on the Trello card](https://trello.com/c/5pr2WcQH/43-diagnostics-additional-pre-flight-checks#comment-58d42e29cd83526685047ea1), since this is a pre-flight check, perhaps it would be better to use something like `cat /proc/os-release` on the ansible host?